src/Compilers/VisualBasic/Portable/Lowering/AsyncRewriter/AsyncRewriter.AsyncMethodToClassRewriter.Await.vb

```diff
@@ -210,12 +210,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         ' STMT:   this.builder.AwaitUnsafeOnCompleted(Of TAwaiter,TSM)((ByRef) $awaiterTemp, (ByRef) Me)
                         '  or
                         ' STMT:   this.builder.AwaitOnCompleted(Of TAwaiter,TSM)((ByRef) $awaiterTemp, (ByRef) Me)
+                        Dim useSiteInfo As CompoundUseSiteInfo(Of AssemblySymbol) = Nothing
```

Comment:
>Dim [](start = 24, length = 3)

Is this intended to be a bug fix?  If so, do we have a test for it?  If yes, please identify that test.

Reply:
> Is this intended to be a bug fix?  If so, do we have a test for it?  If yes, please identify that test.

This is intended as a change to follow a long standing requirement to report use-site errors for all operations/symbols that might have it. We don't have a test for this particular change and I do not believe it is worth spending time coming up with specific scenario demonstrating the problem. It is always safe to report use-site errors, at worse this can lead to a duplicate diagnostic being reported. If you prefer, I can undo the change and open an issue instead. 

---
In reply to: [362285747](https://github.com/dotnet/roslyn/pull/40640#discussion_r362285747) [](ancestors = 362285747)

Reply:
No, I was just wondering.

---
In reply to: [362286535](https://github.com/dotnet/roslyn/pull/40640#discussion_r362286535) [](ancestors = 362286535,362285747)